backend/event-parse/parse.go

```diff
@@ -117,10 +119,26 @@ func (n networkFetcher) fetchStylesheetData(href string) ([]byte, error) {
 		return nil, errors.Wrap(err, "error reading styles")
 	}
 
+	body = replaceRelativePaths(body, href)
 	body = append([]byte("/*highlight-inject*/\n"), body...)
+
 	return body, nil
 }
 
+func replaceRelativePaths(body []byte, href string) []byte {
+	const PROXY_URL string = "https://replay-cors-proxy.highlightrun.workers.dev"
```

Comment:
could this be a top level file constant?

Reply:
Is this preferred?

![image](https://github.com/highlight/highlight/assets/878947/f1bb4361-5e51-49b4-a265-f03c1a216998)

Reply:
Or is it better to put this at the top? 
![image](https://github.com/highlight/highlight/assets/878947/e1da0d48-7cb0-461a-ae03-a63f3cdc0791)

Reply:
I would say at the top of the file with other constants. in go, constants are also typically named with UpperCamelCase ie. `ProxyURL`